### PR TITLE
Fix about page links to gallery

### LINKED ISSUE #58: Broken links on /about
On https://azure.github.io/awesome-azd/about, the three column section with Discover Templates, Create your own, Contrubute to Gallery, contains several broken links.

**To Reproduce**
1. Go to https://azure.github.io/awesome-azd/about
2. Click on the links circled in red below
Screen Shot 2022-10-14 at 11 25 26 AM

**Expected behavior**
I would expect all links circled in red above to go to https://azure.github.io/awesome-azd/

### PR DESCRIPTION
**Pull request by savannahostrowski**
Closes #58

### CODE REVIEW

File: website/src/components/HomepageFeatures.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const FeatureList = [
     link: "https://github.com/Azure/awesome-azd/issues/new/choose",
     description: (
       <>
-        Consider sharing your template in our <a href="/gallery">Gallery</a> to help the community. Just <a href="https://github.com/Azure/awesome-azd/issues/new/choose">fill in the issue </a> and we'll do the rest!
+        Consider sharing your template in our <a href="/awesome-azd">Gallery</a> to help the community. Just <a href="https://github.com/Azure/awesome-azd/issues/new/choose">fill in the issue </a> and we'll do the rest!
```
Comment (nitya) [inline website/src/components/HomepageFeatures.js:40]: the app does not have an "/awesome-azd" route - did you mean to use "/" (so it points to the default homepage for Gallery?)

**Review by nitya — DISMISSED** [review]
Change second link to:
<a href="/">Gallery</a>

The current path "/awesome-azd" does not exist in app.

**Comment by savannahostrowski** [comment]
Both links were broken with `/` on the live site, it just points to `azure.github.io/`. This actually worked locally. Is something misconfigured?

**Review by nitya — APPROVED** [review]
SGTM

**Pull request merged**